crates/geoblock/src/lib.rs

```diff
@@ -0,0 +1,287 @@
+/// Middleware which adds geo-location IP blocking.
+///
+/// Note: this middleware requires you to use
+/// [Router::into_make_service_with_connect_info](https://docs.rs/axum/latest/axum/struct.Router.html#method.into_make_service_with_connect_info)
+/// to run your app otherwise it will fail at runtime.
+///
+/// See [Router::into_make_service_with_connect_info](https://docs.rs/axum/latest/axum/struct.Router.html#method.into_make_service_with_connect_info) for more details.
+pub use geoip;
```

Comment:
I like the idea of re-exporting deps, as it may solve some incompatibilities. In the relay for example, we may use some of the `geoip2` types and features for analytics purposes.

Reply:
Yea, but this is a middleware crate and not a general purpose library. Anyway if both of you want to have it - let's have it :)
Breaking changes won't be a problem for a foreseeable future

Reply:
This is `utils-rs`. We may re-export it from the root, or some other way. But it's a nice place to collect common dependencies  in my opinion.